src/axom/spin/UniformGrid.hpp

```diff
@@ -796,7 +796,7 @@ void UniformGrid<T, NDIMS, ExecSpace, StoragePolicy>::getCandidatesAsArray(
         int count = counts_view[i];
         if(count > 0)
         {
-  #ifndef __CUDA_ARCH__
+  #ifndef AXOM_DEVICE_CODE
```

Comment:
Please confirm that this is not affected by #734

Reply:
> Please confirm that this is not affected by #734

Yes, this was okay. I don't have `mesh_tester` working with hip yet, but testing with CUDA on blueos, and sequential on blueos and toss 4 were fine.